src/components/FormField/Input.tsx

```diff
@@ -0,0 +1,268 @@
+import { ChangeEvent, HTMLAttributes, ReactNode, useState } from "react";
+import { Error, FormRoot, Label } from "./commonElement";
+import styled from "styled-components";
+import { Icon } from "..";
+
+const InputContainer = styled.div<{ error: boolean }>`
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  align-items: center;
+
+  span:first-of-type {
+    max-width: 100%;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+  ${({ theme, error }) => `
+    border-radius: ${theme.click.field.radii.all};
+    font: ${theme.click.field.typography["field-text"].default};
+    color: ${theme.click.field.color.text.default};
+    border: 1px solid ${theme.click.field.color.stroke.default};
+    background: ${theme.click.field.color.background.default};
+    &:hover {
+      border: 1px solid ${theme.click.field.color.stroke.hover};
+      background: ${theme.click.field.color.background.hover};
+      color: ${theme.click.field.color.text.hover};
+    }
+    & > input {
+      padding-top: ${theme.click.field.space.y};
+      padding-bottom: ${theme.click.field.space.y};
+      padding-left: ${theme.click.field.space.x};
+    }
+    ${
+      error
+        ? `
+      font: ${theme.click.field.typography["field-text"].error};
+      border: 1px solid ${theme.click.field.color.stroke.error};
+      background: ${theme.click.field.color.background.active};
+      color: ${theme.click.field.color.text.error};
+      &:hover {
+      border: 1px solid ${theme.click.field.color.stroke.error};
+      color: ${theme.click.field.color.text.error};
+      }
+    `
+        : `
+    &:focus-within,
+    &[data-state="open"] {
+      font: ${theme.click.field.typography["field-text"].active};
+      border: 1px solid ${theme.click.field.color.stroke.active};
+      background: ${theme.click.field.color.background.active};
+      color: ${theme.click.field.color.text.active};
+      &~ label {
+      color: ${theme.click.field.color.label.active};
+      font: ${theme.click.field.typography.label.active};;
```

Comment:
I'm not sure why this token isn't being updated on dark mode, I've checked in Figma, seems to be set up correctly.

<img width="326" alt="CleanShot 2023-07-21 at 08 17 13@2x" src="https://github.com/ClickHouse/click-ui/assets/305167/8d6ffe03-6b85-4215-9597-9c21cd7e4c04">